mlrun/config.py

```diff
@@ -405,17 +409,45 @@ def get_hub_url():
         return config.hub_url
 
     @staticmethod
-    def get_default_function_node_selector():
-        default_function_node_selector = {}
-        if config.default_function_node_selector:
-            default_function_node_selector_json_string = base64.b64decode(
-                config.default_function_node_selector
-            ).decode()
-            default_function_node_selector = json.loads(
-                default_function_node_selector_json_string
-            )
+    def decode_base64_config_and_load_to_dict(attribute_name: str):
+        parts = (
+            attribute_name.split(".")
```

Comment:
I would rename `attribute_name` to `attribute_path` and then here `attributes` instead of `parts`